packages/beacon-node/src/eth1/eth1DepositDataTracker.ts

```diff
@@ -175,16 +175,16 @@ export class Eth1DepositDataTracker {
           await sleep(sleepTimeMs, this.signal);
         }
       } catch (e) {
+        this.metrics?.eth1.depositTrackerUpdateErrors.inc(1);
+
         // From Infura: 429 Too Many Requests
         if (e instanceof HttpRpcError && e.status === 429) {
           this.logger.debug("Eth1 provider rate limited", {}, e);
           await sleep(RATE_LIMITED_WAIT_MS, this.signal);
+          // only log error if state switched from online to some other state
         } else if (!isErrorAborted(e)) {
-          this.logger.error("Error updating eth1 chain cache", {}, e as Error);
```

Comment:
@nazarhussain can you please explain this change, does the comment related to the removed log?

Is the log on line 183 supposed to be replaced by the error log added in the eth1 provider?
https://github.com/ChainSafe/lodestar/blob/4c09412ed2c8a94b585d5dba1fe94bc9e2d56a35/packages/beacon-node/src/eth1/provider/eth1Provider.ts#L112

Reply:
Yes this log line should be replaced with eth1Provider error handling, but because we log the error once per minute so it could be the case you don't see the errors often.